docs/channel.md

```diff
@@ -80,33 +80,27 @@ See {ref}`channel-factory` for the full list of channel factories.
 
 ## Operators
 
-Operators are methods that consume and produce channels. Because channels are asynchronous, operators are necessary to manipulate the values in a channel, without using a process. As a result, operators are useful for implementing the "glue logic" between processes.
+Channel operators, or "operators" for short, are functions that consume and produce channels. Because channels are asynchronous, operators are necessary to manipulate the values in a channel, aside from using a process. As a result, operators are useful for implementing the "glue logic" between processes.
```

Comment:
```suggestion
Channel operators, or _operators_ for short, are functions that consume and produce channels. Because channels are asynchronous, operators are necessary to manipulate the values in a channel, aside from using a process. As a result, operators are useful for implementing the _glue logic_ between processes.
```
We are moving towards using italics for words/phrases that are quoted.

Reply:
NIce